### DISCUSSION TITLE
MWC 2024 Ticket Giveaway!

### DISCUSSION THREAD

**Discussion by brendan-mcauliffe** [srsran/srsRAN_Project#466] [Announcements]
🎉📱 Exciting News Alert! 📱🎉

Are you doing interesting work with srsRAN?
Would you like to attend the biggest annual event in telecommunications?

Mobile World Congress takes place in Barcelona this month from February 26th to 29th and we have two tickets to give away to srsRAN users. 🇪🇸✨

Just reply to this post with a brief outline of your work and confirming your availability to attend. Tickets will be allocated early next week. Be creative! We love hearing about your wild ideas and ambitious projects! 🚀💡

That's it, good luck!

**Comment by improperaffo** [comment]
HI! We are very interested in the MWC, we are two students at EPFL (Switzerland), we have been working with the project for almost a year. In the past 6 month we got srsRAN_Project to work with BladeRF and OrangePI 5.
Currently we have split the work: @HaoxinSEU is working on getting FlexRIC to interact with the gNB, and I am working on the slice support.

This is our class break and we are not far from Barcelona, so we have the time to come and meet you.

These are some contributions of ours over the last 6 months:
https://github.com/srsran/srsRAN_Project/discussions/222
https://github.com/srsran/srsRAN_Project/discussions/339
https://github.com/srsran/srsRAN_Project/discussions/219
https://github.com/srsran/srsRAN_Project/discussions/246
https://github.com/srsran/srsRAN_Project/issues/288
https://github.com/srsran/srsRAN_Project/issues/281
https://github.com/srsran/srsRAN_Project/discussions/290

**Comment by j0lama** [comment]
Hello there! My name is Jon Larrea, and I'm a final-year PhD student at the University of Edinburgh. My research began in the core network, where one of the initial deployments utilized srsEPC and srsRAN. Since then, I've been heavily involved in the automation and cloudification of the RAN, integrating srsRAN_4G and srsRAN_Project with Kubernetes. More recently, I've been developing various radio sniffers and decoders using srsRAN as the foundational tool, which we will soon release.

P.S.: I'll be visiting my family in Spain during the latter half of February :smiley:

**Comment by brendan-mcauliffe** [comment]
Thanks for entering, the tickets have been assigned and we will be closing the discussion.
